src/style/scheme.ts

```diff
@@ -7,9 +7,10 @@ export type SchemeType = 'default' | 'light' | 'dark';
 export type SchemeChangeListener = (schemeType?: 'light' | 'dark') => void;
 
 class Scheme {
-  currentScheme: SchemeType = 'default';
-  schemes: Schemes = {light: {}, dark: {}};
-  changeListeners: SchemeChangeListener[] = [];
+  private currentScheme: SchemeType = 'default';
+  private schemes: Schemes = {light: {}, dark: {}};
+  private schemesJS: Schemes = {light: {}, dark: {}};
```

Comment:
I don't understand how `schemesJS` is different from `schemes`... why we need the two?

Reply:
Did you look how do we store each in `loadScheme` method?

Reply:
Yes

Reply:
Basically in `schemes` we keep the native platformColors 
The values that are kept there are not regular strings but a weird object that the native side can handle

That's the reason I also want to keep the plain JS strings in `schemesJS` 
It should serve the `Colors.getColorValue` method when we need to get the original JS string hex value in some cases. 

Why do we need the JS string value? 
One of the cases, is when we call the `Colors.getColorTint` in the private repo. 
With the recent PlatformColor support, this method throws an error because `Colors.getColorTint` expect a string but now gets a weird object. 
So this whole PR meant to let us get the original string value so we can pass to `Colors.getColorTint`

I hope this explains better

Reply:
Sorry I wasn't clear with my question. I understand the meaning behind the pr and why we want to keep the js schemes. What I was missing is where are you loading these schemesJS with the string values.

Reply:
Anywhere we call `Colors.loadSchemes` (It calls internally `Scheme.loadSchemes`) and it always accept a maps of strings colors. 
We internally mapped them once to platform colors or strings

Reply:
Ok